packages/app-backend-core/src/legacy/scan.ts

```diff
@@ -60,6 +61,17 @@ export function scan () {
         // Ignore
       }
     }
+
+    // Scan for Vue instances in the customTarget elements
+    const { customVue2ScanSelector } = getPageConfig() as { customVue2ScanSelector: string }
```

Comment:
Could you add the field to the `PageConfig` interface instead of casting?
![image](https://user-images.githubusercontent.com/2798204/148406778-93e8eaf8-1dcd-4923-b92d-aad620fcbad7.png)